# Logfile created on 2014-01-29 11:42:19 -0300 by logger.rb/v1.2.7
*** Below you'll find the most recent exception thrown, this will likely (but not certainly) be the exception that made your application exit abnormally ***
#<Interrupt: Thread has been interrupted>
*** Below you'll find all the exception objects in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
#<Interrupt: Thread has been interrupted>
